Packages/ANCIENT-MEGALITHIC-MYSTERY-PACKAGE/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="atg-location" content="Cusco">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="../../css/main.css">
+	<link rel="stylesheet" type="text/css" href="../../css/global.css">
+	<!-- PACKAGE STYLES -->
+	<link rel="stylesheet" type="text/css" href="../../css/packages.css">
+	<title>ANCIENT MEGALITHIC MYSTERY PACKAGE</title>
+	<!-- Script for generating the data -->
+	<script src="../../js/Packages.js"></script>
+</head>
+<body>
+	<div class="first Cusco">
+		<div class=logo>
+			<img src="https://res.cloudinary.com/dlia7phdx/image/upload/v1564095577/ATG/atg-logo.png">
```

Comment:
use the picture tag - also don't use png - use the SVG please... using the picture tag, you can set media query to fetch varying sizes or versions - for example, for svg (which scales nicely), you can choose a small version of the logo that might not display text bylines that would otherwise be too small to read anyway

Reply:
ok ill change it right now